### PR TITLE
fix: refine wrong DOM sanitization

### PR DESCRIPTION
**Pull request by adhrinae**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

- This PR fixes the wrong implementation of the XSS prevention in #879.

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/js/controller/base.js [hunk 1]
```diff
@@ -13,6 +14,23 @@ var Theme = require('../theme/theme');
 var tz = require('../common/timezone');
 var TZDate = tz.Date;
 
+var SCHEDULE_VULNERABLE_OPTIONS = ['title', 'body', 'location', 'state', 'category', 'dueDateClass'];
```
Comment (dongsik-yoo) [inline src/js/controller/base.js:17]: title, body외에 다른 값도 html 렌더링에 사용되는지 확인해 봐야겠습니다.

**Review by dongsik-yoo — COMMENTED** [review]
리뷰 완료합니다. 수고하셨습니다.

File: src/js/controller/base.js (hunk 1 above)
Comment (adhrinae) [reply src/js/controller/base.js:17]: 위의 값들은 최소 디테일 팝업에 랜더링 될 여지가 있는 값으로 보여 추가했습니다.

**Review by jwlee1108 — APPROVED** [review]
리뷰 완료합니다.

**Review by lja1018 — APPROVED** [review]
리뷰완료합니다.

**Pull request merged**